src/components/CardBox/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { useContext } from 'react'
+import { CardInfoContext } from '../../context/paymentContext'
+import ui from '../../styles/index.module.css'
+export const CardBox = () => {
```

Comment:
```suggestion
import { useContext } from 'react'
import { CardInfoContext } from '../../context/paymentContext'
import ui from '../../styles/index.module.css'

export const CardBox = () => {
```

import문과 모듈 내부 코드, 관심사 그루핑이 필요한 부분 등에 공백을 이용해서 코드들을 잘 분리하시면 읽는 사람 입장에서 더 읽기 수월해질거에요.

잘해주시고 있는 부분들이 많은데 코드 중간중간 통일되지 않은 부분들이 있어서 지연님께서는 평소 어떤식으로 하시고 계셨는지 스스로 고민해보시고 코드 작성시 의식적으로 공백에도 신경써주시면 좋을 것 같아요!

```
import Something from './someModule'
type SomeProp = { something: string; }
export const SomeComponent = () => {
    const [state, setState] = useState();
    const handleSomeEvent = () => {}
    useEffect(() => {},[])
    return (<div/>)
}
```

```
import Something from './someModule'

type SomeProp = { something: string; }

export const SomeComponent = () => {
    const [state, setState] = useState();

    const handleSomeEvent = () => {}

    useEffect(() => {},[])

    return (<div/>)
}
```
저는 위 코드보다는 아래코드가 더 보기에 좋다고 생각해요 🧐